index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TochukwuAndMarioPage</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media (max-width: 600px) {
+        .facet_sidebar {
+          display: none;
+        }
+      }
+      </style>
```

Comment:
I suggest you set styles on CSS document instead of doing in HTML. It's a good and professional practice.